src/constants.js

```diff
@@ -0,0 +1,2 @@
+const API_BASE_URL = 'https://api.realdevsquad.com';
+const userManagementBtn = document.getElementById('user-management-btn');
```

Comment:
Constants should not contain a reference to a live DOM object. What if this object is removed using JS in future somewhere else. This reference will become orphaned and wrong.

Reply:
ok, but once @Neha suggested that UI elements should also be in the constants file, in this issue it's mentioned(https://github.com/Real-Dev-Squad/website-dashboard/issues/203), though she suggested there should be different constants files for UI elements but for that we need to restructure the file structure(which I am working on for different issues), so, for now, I kept all the types constants in this file only.
Let me know if i understood something wrong